lucene/translate.py

```diff
@@ -154,36 +169,43 @@ def postprocess_translated_objc(path):
             if any(fnmatch.fnmatch(full_path_java, ptn) for ptn in excluded):
                 continue
 
-            full_path_m = full_path_java.replace(
-                src, dst).replace(".java", ".m")
+            full_path_m = full_path_java.replace(src, dst).replace(".java", ".m")
             if os.path.exists(full_path_m):
                 if os.path.getmtime(full_path_m) >= os.path.getmtime(full_path_java):  # nopep8
                     continue
+
             to_compile.append(full_path_java)
             to_postprocess.append(full_path_m)
 
-    print('%s => %s files to compile' % (src, len(to_compile)))
+    print('\nTranslating: %s with %d java files to compile.\n' % (src, len(to_compile)))
 
     if to_compile:
-        print('Compiling %d java files' % len(to_compile))
         args = [
-            'j2objc',
-            # '-use-arc',
-            '-classpath', classpaths,
-            '--segmented-headers',
-            '-sourcepath', src,
+            j2objc,
             '-d', dst,
+            '-classpath', ':'.join(classpaths),
+            '--swift-friendly',
```

Comment:
Out of curiosity (and I assume you intend to write your app in Swift and call the Objective-C Lucene directly), do you notice any difference if you don't use `--swift-friendly` or `--nullability`?

The same question for `--no-extract-unsequenced`. Does that affect the way you use Lucene in your app?

Good call to add `--doc-comments`. I usually switch between Xcode and IntelliJ for this, but I guess this saved some context switching especially if you focus on Objective-C / Swift. :)